nondex-common/src/main/java/edu/illinois/nondex/common/Configuration.java

```diff
@@ -55,6 +55,7 @@ a copy of this software and associated documentation files (the
 
     public final long start;
     public final long end;
+    public final String log;
```

Comment:
What log is this?
log -> systematicLog
The names of your variables should be descriptive. Imagine that there may be other logs that the tool creates, e.g., a debug log.